lib/omsmailer_web/templates/page/candidate_applied.html.eex

```diff
@@ -0,0 +1,82 @@
+            id="contentTable">
+            <tbody>
+              <tr>
+                <td width="600" valign="top" align="center" style="border-collapse:collapse;">
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="background: #F9FAFB;" width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;">&nbsp;</div>
+                        </td>
+                      </tr>
+                    </tbody>
+                  </table>
+                  <table align="center" border="0" cellpadding="0" cellspacing="0" style="border: 1px solid #E0E4E8;"
+                    width="100%">
+                    <tbody>
+                      <tr>
+                        <td align="center" style="padding: 56px 56px 28px 56px;" valign="top">
+                          <a style="color: #3498DB; text-decoration: none;" href="my.aegee.eu"
+                            target="_blank"><img style="width: 56px; height: 56px; border: 0;" alt="Logo"
+                              src="https://my.aegee.eu/images/logo.png">
+                          </a></td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 20px; color: #333;"><strong>A new application is submitted</strong></div>
+                          <div style="font-family: &quot;lato&quot;, &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; line-height: 28px;font-size: 16px; color: #333;">
+                            A new application to position
+                            <strong><%= @parameters["position"]["name"] %></strong>
+                            is submitted by
+                            <strong><%= @parameters["candidate"]["first_name"] @parameters["candidate"]["last_name"] %></strong>.
+                          </div>
+                        </td>
+                      </tr>
+                      <tr>
+                        <td align="center" style="padding: 0 56px 28px 56px;" valign="top">
+                          <div>
+                            <a href="https://my.aegee.eu/statutory/<%= @parameters["position"]["event_id"] %>/positions/all" style="background-color:#1468C5;border-radius:50px;color:#ffffff;display:inline-block;font-family: 'lato', 'Helvetica Neue', Helvetica, Arial, sans-serif;font-size:18px;font-weight: bold;line-height:40px;text-align:center;text-decoration:none;width:270px;-webkit-text-size-adjust:none;"
```

Comment:
Is event_id a string? Or does the numerical id also work for the link?

Reply:
It is a numerical id, and that also works for the link (apparently since I just had to check it 😅 )

Reply:
yep, Leon is right